Homework/Homework_10/cache_func.py

```diff
@@ -0,0 +1,22 @@
+def cache(func):
+    cache_dictionary = {}
+
+    def wrapper(arg):
+        if arg in cache_dictionary:
+            return cache_dictionary[arg]
+        cache_dictionary[arg] = func(arg)
+        return func(arg)
```

Comment:
```suggestion
        return cache_dictionary[arg]
```

Reply:
you do not need to call the same function twice

Reply:
Done